test_fatoptimizer.py

```diff
@@ -3225,6 +3225,29 @@ def test_str(self):
         self.check_optimize(r'" ".isspace()', 'True')
         self.check_optimize(r'"AbC".swapcase()', '"aBc"')
         self.check_optimize(r'"hello world".title()', '"Hello World"')
+        self.check_optimize(r'"abc".center(5)', '" abc "')
+        self.check_optimize(r'"hello".count("l")', '2')
+        self.check_optimize(r'"abc".endswith("c")', 'True')
+        self.check_optimize(r'"01\t012\t0123\t01234".expandtabs()', '"01      012     0123    01234"')
+        self.check_optimize(r'"Python".find("Py")', '0')
+        self.check_optimize(r'"Python".index("Py")', '0')
+        self.check_optimize(r'"jgh\ffh".isprintable()', 'False')
+        self.check_optimize(r'"ABC".isupper()', 'True')
+        self.check_optimize(r'"ABC".ljust(5)', '"ABC  "')
+        self.check_optimize(r'"   spacious   ".lstrip()', '"spacious   "')
+        self.check_optimize(r'"hello".maketrans("l","d")', '{108: 100}')
+        self.check_optimize(r'"hello".partition("l")', '('he', 'l', 'lo')')
```

Comment:
Did you try to run tests?

```
python3 test_fatoptimizer.py
```

Or better:

```
tox -e py3
```

(You may have to install tox: sudo python3 -m pip install -U tox)